### PR TITLE
Add Merchant Verified Page

### PR DESCRIPTION
**Pull request by vahinim**
Add merchant_verified_view.dart that displays Merchant is already verified.
Whenever Agent tries to verify Store, that is already verified it displays this view.
Update fetch_store.dart to include this information.
Update globals to store status.
Add store_status.dart that fetches the store status information.

### CODE REVIEW

File: frontend/gpay_agent/lib/agent_datamodels/store_status.dart [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+import 'package:agent_app/globals.dart' as globals;
+
+/// Store model that stores the Store fetched from spanner database.
```
Comment (trblessy) [inline frontend/gpay_agent/lib/agent_datamodels/store_status.dart]: A lot of "store"s in one sentence :)

File: frontend/gpay_agent/lib/agent_datamodels/store_status.dart [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    if (response.statusCode == 200) {
+      print('The Store is verified');
+      try {
+        LineSplitter lineSplitter = new LineSplitter();
+        List<String> lines = lineSplitter.convert(response.body);
+        String jsonString = lines[0];
+        globals.storeStatus = StoreStatus.fromJson(jsonDecode(jsonString));
+        if(globals.storeStatus.status == "green" || globals.storeStatus.status == "red"){
+          globals.isStoreVerified = true;
+        }else{
```
Comment (trblessy) [inline frontend/gpay_agent/lib/agent_datamodels/store_status.dart]: } space else space {

Comment (trblessy) [reply frontend/gpay_agent/lib/agent_datamodels/store_status.dart]: using the formatting tool should help you with this

File: frontend/gpay_agent/lib/agent_datamodels/store_status.dart (hunk 1 above)
Comment (vahinim) [reply frontend/gpay_agent/lib/agent_datamodels/store_status.dart]: updated

File: frontend/gpay_agent/lib/agent_datamodels/store_status.dart (hunk 2 above)
Comment (vahinim) [reply frontend/gpay_agent/lib/agent_datamodels/store_status.dart]: reformatted

**Review by trblessy — APPROVED** [review]
Make sure all the tests pass before merging this PR

**Pull request merged**